sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_promptflow/azure/__init__.py

```diff
@@ -0,0 +1,3 @@
+# ---------------------------------------------------------
```

Comment:
Can we have the azure folder without the parent promptflow folder? I think we want to gradually phase out the association with Promptflow in the AI Evaluation SDK.